### ISSUE TITLE
Content support on Ask VA category labels

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#63280]
Ask VA team provided [this spreadsheet](https://docs.google.com/spreadsheets/d/1aCei8pmLBBkWPeDye4YSmUqRkD0yCfTeLEqwdL-329M/edit?usp=sharing) of proposed label changes for the categories and subcategories in Ask VA. 

We agreed with them that:
- We'll edit directly in the spreadsheet to align labels with site style and consistency with live pages
- We'll look at the top-level categories, caregiver support, and education first
- They're meeting with additional business lines on a rolling basis
- We'll also suggest a PL approach to labels at the form question level for "subcategory", etc.

Here's the planned order to prioritize our label work:
- Debt
- Life insurance
- Veteran ID cards
- Resources for Women Veterans
- Sign in and technical issues
- DEERS
- Center for Minority Veterans
- Decision reviews and appeals
- Veterans living abroad
- Disability compensation
- Pension
- Fiduciary program
- Survivor benefits
- Burials and memorials
- Veteran Readiness and Employment
- Housing assistance

**Comment by tygindraux** [comment]
And we've already reviewed these with the business lines:
- Health
- Education

Sorry, I should have included those top of the list!

**Comment by tygindraux** [comment]
@laurwill I added two more columns to the [spreadsheet](https://docs.google.com/spreadsheets/d/1aCei8pmLBBkWPeDye4YSmUqRkD0yCfTeLEqwdL-329M/edit#gid=0), to make it easier to track feedback:

- Comments: CAIA
- Comments: Business line

We've started receiving feedback from business lines on proposed names and I'm hoping we can document their feedback alongside yours. This way you can see their feedback when you review the spreadsheet, if we've already met with that business line (and v.v. we can see your feedback when we review the business line if we haven't met with them already).

**Comment by ruchishah29** [comment]
@laurwill - could you please provide a status on this?

**Comment by tygindraux** [comment]
Hi @laurwill! Thanks again for this mammoth task.

I know you're working with Danielle and Tom Black (Identity) to clean up the `Sign in and technical issues` category. There are also 3 more topics in the `Education benefits and work study` category that need a final review.

I'm hoping to confirm when you'll have these updates passed over to our team, especially since this is blocking some other tickets for us. Does 10/30 still seem realistic?

Thanks again!

cc: @ruchishah29

**Comment by laurwill** [comment]
Hi @tygindraux @ruchishah29 , 

Here's the [label sheet](https://docs.google.com/spreadsheets/d/1qRpAwE-OmISFzYsg4w3n8VlBsxJxkBAFZL21AWDXFKc/edit#gid=1687719720) back with our final copyedit changes. 

And here are a few notes:
- We created new columns for the copyedit — copy edit changes, and copy edit notes. This way, you can see what we changed before adjusting the actual labels. 
- Just reiterating that for the most part, this round included more surface-level language changes. We'll still want to support the more substantive taxonomy and restructuring work in the next round of the integration project.
- Signing in and technical issues — I think we need to walk back the creation of the new "signing in to other websites" topic. There aren't enough subtopics to populate this in a way that makes sense, so I think better to wait until the next phase of this work when we'll be doing more taxonomy and restructuring of categories. After speaking with Danielle, we made language tweaks to the signing in category for now, with the understanding that this one will need more substantive changes in the next phase.
- Debt — This one had open questions in the original sheet that we never resolved. Currently, we've adjusted some but not all of the topics to "xyz benefit overpayments" because we were waiting for confirmation from the business line that all the topics except for copay debt relate to overpayments. We shouldn't leave a mix of "overpayments" and "debt" in the topic list for this category, as that could be misleading. Suggest asking business line to confirm the questions I left in the sheet. (If I had to guess, "copay debt" is the only one that isn't technically an overpayment. But some of these are unfamiliar terms, so we need to confirm.)

**Comment by tygindraux** [comment]
Hi @laurwill, thank you for the update!

- Copyedit columns: Noted! This is clear.
- More substantive taxonomy and restructuring work in the next round: Agreed.
- Signing in and technical issues: No problem, I'll remove the new topic from the final list. And noted that we'll look more closely at this again after Phase 1.
- Debt: Thanks for catching these open questions, it looked like they slipped through. I'll go back to the business line this week to confirm the questions and then let you know where we land.

Thanks again for all of your and your team's work on this.

**Comment by laurwill** [comment]
I suggested an update for a label related to [this ticket on Ask VA team's board](https://app.zenhub.com/workspaces/ask-va-647a476551689d06655cc815/issues/gh/department-of-veterans-affairs/ask-va/826). 

I'm going to close this out since we've completed the first pass at these labels — @tygindraux I can make one-off updates as needed between now and MVP launch, and then we can open new tickets for the post-MVP work. Thanks!
